ui/component/channelSelector/view.jsx

```diff
@@ -139,18 +160,25 @@ type ListItemProps = {
   claimsByUri: { [string]: any },
   doFetchUserMemberships: (claimIdCsv: string) => void,
   odyseeMembershipByUri: (uri: string) => string,
+  isPublishMenu?: boolean,
+  isTabHeader?: boolean,
 };
 
 function ChannelListItem(props: ListItemProps) {
-  const { uri, isSelected = false, claimsByUri, doFetchUserMemberships, odyseeMembershipByUri } = props;
+  const { uri, isSelected = false, claimsByUri, doFetchUserMemberships, odyseeMembershipByUri, isPublishMenu } = props;
 
   const membership = odyseeMembershipByUri(uri);
 
   const shouldFetchUserMemberships = true;
   useGetUserMemberships(shouldFetchUserMemberships, [uri], claimsByUri, doFetchUserMemberships, [uri]);
 
   return (
-    <div className={classnames('channel__list-item', { 'channel__list-item--selected': isSelected })}>
+    <div
+      className={classnames('channel__list-item', {
+        'channel__list-item--selected': isSelected,
+      })}
+    >
+      {isPublishMenu && <div className="channel__selector--prefix">{__('as')}</div>}
```

Comment:
- [ ] Can't translate words standalone or split-strings because it wouldn't make sense in some languages.
- [ ] The left-to-right nature of "sentence" flow is also problematic since some languages is in the inverse.

Usually, for strings, we let the translators decide the word order, but since this also involves button order (design), it gets a bit tricky. 

Maybe just drop the 'as' and assume they know what's going on.

Aside: (Just 1 person's opinion, not request for change): it feels a bit inversed that the URL component at the top depends on a drop-down value at the bottom.  The flow with the drop-down at the top feels more natural, although it doesn't fit the overall design.

Reply:
Yeah, I've mentioned the issue with the label order during our last meeting. I've noticed the same issue when I switched to Japanese. Removing the label should be fine for now.

I've switched the position of the dropdown because during my tests I often uploaded and edited things with the wrong user account. The problem is, that by the time you reach the bottom of the form and are about to click the final button you can no longer see the publishing user (except for the confirmation modal, but that can be disabled). That's why I moved it to the bottom. I think it's the very last thing you should see before you hit that last button. Valid point regarding the url field though.